Grammar (#76)
fhdsl/NIH_Data_Sharing: Appendix.Rmd

Feel free to remix these documents to suit your needs with credit to the NIH DMSP Guidance Working Group.

> In addition, there are several tools and resources that can help you write your plan and make sure that it complies with all the policies that you need to comply with:

Reviewer: ```suggestion
In addition, there are several tools and resources that can help you write your plan while complying with NIH policies:
```

Author: hmmm since some of them are not actually for writing I think "and" might be better - also for lots of policies beyond NIH

Author: I can still shorten it though

Author: how's that? I edited your suggestion @avahoffman
